client-sdk/ts-web/core/src/client.ts

```diff
@@ -1115,65 +1085,13 @@ export class NodeInternal extends GRPCWrapper {
         return this.callServerStreaming(methodDescriptorStakingWatchEvents, undefined);
     }
 
-    /**
-     * GetEphemeralSecret returns the key manager ephemeral secret.
-     */
-    keyManagerGetEphemeralSecret(query: types.RegistryNamespaceQuery) {
-        return this.callUnary(methodDescriptorKeyManagerGetEphemeralSecret, query);
-    }
-
-    /**
-     * GetMasterSecret returns the key manager master secret.
-     */
-    keyManagerGetMasterSecret(query: types.RegistryNamespaceQuery) {
-        return this.callUnary(methodDescriptorKeyManagerGetMasterSecret, query);
-    }
-
-    /**
-     * GetStatus returns a key manager status by key manager ID.
-     */
-    keyManagerGetStatus(query: types.RegistryNamespaceQuery) {
-        return this.callUnary(methodDescriptorKeyManagerGetStatus, query);
-    }
-
-    /**
-     * GetStatuses returns all currently tracked key manager statuses.
-     */
-    keyManagerGetStatuses(height: types.longnum) {
-        return this.callUnary(methodDescriptorKeyManagerGetStatuses, height);
-    }
-
```

Comment:
were these intentionally removed?

Reply:
Yes. With CHURP the key manager actually has multiple sub-modules, "secrets" and "churp".